### DISCUSSION TITLE
Date table column translations

### DISCUSSION THREAD

**Discussion by mrSkovgaard** [sql-bi/Bravo#563] [Q&A] [answered]
My end users requires that the columns in the Date table must be in Danish. What is the recommended approach to achieve this?

**Comment by albertospelta** [comment, marked as answer]
You can fork the Bravo repo and submit a new pull request updating the `DateLocalization` JSON files containing the strings used in the Dates templates as described here https://github.com/sql-bi/Bravo#dates-template-translations.

You can take a look at this PR https://github.com/sql-bi/Bravo/pull/378/files which added Chinese translation for date table columns.

**Reply by mrSkovgaard** [reply]
Thanks. As suggested I have added Danish translation and made a pull request.
